hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/webapp/dao/CapacitySchedulerInfo.java

```diff
@@ -88,6 +90,9 @@ public CapacitySchedulerInfo(CSQueue parent, CapacityScheduler cs) {
     health = new CapacitySchedulerHealthInfo(cs);
     maximumAllocation = new ResourceInfo(parent.getMaximumAllocation());
 
+    isAbsoluteResource = parent.getCapacityConfigType() ==
+        AbstractCSQueue.CapacityConfigType.ABSOLUTE_RESOURCE;
+
```

Comment:
Question about this: do we actually need this? Because we have the mode property below, which is suggested to be used.

Reply:
I think the mode property is superior to the isAbsoluteResource flag, the latter was introduced before the mode property AFAIk. I just added to the root level for consistency, if someone wants to build something on the isAbsoluteResource property. The mode property is available on root level too.

So it's just for consistency. But if not needed I can remove this change, that's why I added to a separate commit.

Reply:
Fair point, I agree with the addition.